src/ReachSets/ContinuousPost/TMJets/post.jl

```diff
@@ -1,6 +1,68 @@
 using TaylorSeries: set_variables
 using LazySets.Approximations: box_approximation
 
+import TaylorModels
+import IntervalArithmetic
+const IA = IntervalArithmetic
+const zeroI = IA.Interval(0.0) # [0.0, 0.0]
+const oneI = IA.Interval(1.0) # [1.0, 1.0]
+const symI = IA.Interval(-1.0, 1.0)
+@inline zeroBox(m) = IntervalBox(zeroI, m)
+@inline symBox(m) = IntervalBox(symI, m)
+
+function _to_hyperrectangle(tTM, xTM, n)
+    N = length(xTM)
+    SET_TYPE = Hyperrectangle{Float64, SVector{n, Float64}, SVector{n, Float64}}
+    Rsets = Vector{ReachSet{SET_TYPE}}(undef, N-1)
+    @inbounds for i in 1:N-1
+        Hi = convert(Hyperrectangle, xTM[i])
+        t0 = tTM[i]; t1 = tTM[i+1]
```

Comment:
I prefer to not have multiple statements per line because they make the code shorter but harder to read (at least for me).